pkg/client/src/app/pages/repositories/Mvn.tsx

```diff
@@ -107,9 +114,10 @@ export const RepositoriesMvn: React.FunctionComponent = () => {
   }, [refreshMvnForcedSetting]);
 
   const { volumes, refetch } = useFetchVolumes();
+  const { tasks } = useFetchTasks();
   const [storageValue, setStorageValue] = useState<string>();
   const [currCleanId, setCurrCleanId] = useState<number>(0);
-  const [isCleanDisabled, setIsCleanDisabled] = useState<boolean>(true);
+  const [isCleanDisabled, setIsCleanDisabled] = useState<boolean>(false);
```

Comment:
If the only time `isCleanDisabled` is true is while the `isLoading` property of the mutation is also true, we can probably consider this redundant and remove it and let `isLoading` be the only thing that disables the clean button right?

Edit: ah, I see now that `isLoading` isn't what we want here because that's the polling state, not the entire process of waiting for the clean to finish. Hmm.